src/scss/packages/_required.scss

```diff
@@ -4,7 +4,21 @@
 @import '../uswds-theme/color';
 @import '../uswds-theme/utilities';
 @import '../uswds-theme/components';
-@import '../uswds/packages/required';
```

Comment:
The changes here expand out the contents of [the `_required.scss` package](https://github.com/uswds/uswds/blob/develop/src/stylesheets/packages/_required.scss) to be able to inject the override immediately following [`_functions.scss`](https://github.com/uswds/uswds/blob/823b7a69af6fecce27b0c1de9dc950c146f4ddf4/src/stylesheets/packages/_required.scss#L12), because the luminance function is used later int he same `_required.scss` through [`placeholders/_all.scss`](https://github.com/uswds/uswds/blob/823b7a69af6fecce27b0c1de9dc950c146f4ddf4/src/stylesheets/packages/_required.scss#L17).